### ISSUE TITLE
Add more detail to the CoC

### ISSUE THREAD

**Issue by samdmarshall** [macadminsdotorg/codeofconduct#1]
This section of the CoC should be defined more explicitly as to what the terms means.
<blockquote>Harassment, hate speech, bullying, sea-lioning, and gas-lighting are prohibited activities and are grounds for suspension from the community when there is clear documentation in public channels. If you experience any of these behaviors in direct messages please immediately contact the administrative team.</blockquote>

Currently there is no definition or way to lookup these terms without going to an external source, and to avoid conflict around what constitutes the proper definition for any of these I think they should be defined as part of the CoC.

Additionally there isn't a record of the administrative team in the CoC, so asking someone to contact the team poses a bit of a challenge. 

---

The following section:
<blockquote>Be welcoming of everyone: discrimination based on gender identity, race, religion, nationality, political beliefs, sexual preference, preferred technological ecosystem, and concerns that have not been included in this list will be addressed with progressive discipline based on the severity of the infraction as determined by the administrative team.</blockquote>
Could do a better job of explaining what to repercussions around any form of discrimination listed here. I think it is important to qualify how discrimination leads to harassment.

I think these changes would make for a safer and more professional environment for the community.

**Comment by quovadimus** [comment]
For reference, here is the text from the #ask-the-slack-admins that began this discussion:
(https://macadmins.slack.com/archives/ask-the-slack-admins/p1440280506000811)

alwaysdns [5:55 PM] 
I’m trying to take a larger view of the community and what happens in it. In doing so, I’ve read over (for the billionth time) the CoC. It’s written with a lot of jargon…trendy language. In trying to understand more fully, I've investigated the language use more thoroughly. I've asked a number of my friends of all ages outside of the Mac community (some in tech, some not). Thinking that it may be me - that I'm just too old to know the terms I also asked my children (ages 21-27, so you don't think I'm asking tiny kids) if they know these terms. What I found was that none of the people I interviewed have heard them. 
I’ve been told by my activist friends that it’s called Social Justice language. These phrases would be ok if everyone was part of that subculture. But most of us (not most of us vocal people, but most of the 1000 people) here aren’t part of that community. And because it sways to the political side of life, there might be a better - less isolating - way to say what we want to say.

In order to be more inclusive, I would like to see one of two things happen: either the CoC be rewritten not using those Terms of Art or include definitions that clearly explain those phrases to everyone. Nobody who’s a member of the MacAdmins team should have to go to urban dictionary to understand what is written in the CoC and it should not be threatening. It should include a method to lodge a complaint and it should outline how that complaint will be handled. It should be something that improves the community and not one that simply threatens the members. This document should pass muster with any attorney and this one just doesn't.

It can be a better document. This is a great community and a new member's first vision of it should be welcoming even though it's outlining how not to be a dick. (edited)

macmule [5:57 PM] 
@alwaysdns: Thanks for taking the time to review the CoC. Tbh, I didn't get some of the wording but assumed that was me. So, something to work on.

macmule [5:58 PM]
Really, really appreciate it though & am sure we'll take it on board. Thanks again!!

alwaysdns [5:58 PM] 
my pleasure. always want to see a great community be great in all ways
